plopComponents/application/prompts.js

```diff
@@ -1,111 +1,81 @@
-const vectaraWebsiteQuestions = require(`${__dirname}/../../sampleData/vectara-website/queries.json`);
-const vectaraDocsQuestions = require(`${__dirname}/../../sampleData/vectara-docs/queries.json`);
-const askFeynmanQuestions = require(`${__dirname}/../../sampleData/ask-feynman/queries.json`);
-
-const DEFAULT_CONFIGS = {
```

Comment:
Do we need to provide more than one set of sample data? Seems like one good one is enough for folks to get a sense of the user experience.

Reply:
If we offer more than one sample app, then yes we do. Each sample app's corpus is significantly different that these sample questions may/may not return anything if used for an app they were not originally meant for. But it looks like there's only one sample app now, so the docs one should work fine.